### PR TITLE
Meeting C++ 2018 trip report

### PR DESCRIPTION
**Pull request by jgsogo**
💪  Please, check it and suggest, @conan-io/barbarians @conan-io/community-leaders

### CODE REVIEW

File: _posts/2018-11-21-MeetingCpp-trip-report.markdown [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+comments: false
+title: "Meeting C++ and Meeting Embedded trip report"
+---
+
+
+## Meeting Embedded
+
+On Wednesday, before Meeting C++, we (Diego and Dani) attended 
+and presented a talk at [Meeting Embedded](https://meetingembedded.com/2018/), 
+a new conference about all embedded things. C++ and C had a relevant role in this
```
Comment (jgsogo) [inline _posts/2018-11-21-MeetingCpp-trip-report.markdown]: @danimtb, we need to change that "all embedded things", don't we? Maybe "systems"? "devices"?

Comment (danimtb) [reply _posts/2018-11-21-MeetingCpp-trip-report.markdown]: I think it refers to the fact that the talks were about different topics, from C++ reference or communication protocols to how to make workshops with this devices.

So I'd say somehitng like:

`"...new conference about any topic related to embedded systems."`

File: _posts/2018-11-21-MeetingCpp-trip-report.markdown [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    <figure>
+        <img src="{{ site.url }}/assets/post_images/2018-11-21/lippincot.png" width="600"/>
+        <figcaption>There are no demons, just other players trying to win their games</figcaption>
+    </figure>
+</div>
+
+#### Cross-platform C++ development is challenging - let tools help! (Marc Goodner)
+
+As one would expect, we were really interested in a talk around tooling for the 
+C++ ecosystem. Mac Goodner from Microsoft introduced some of the improvements
```
Comment (danimtb) [inline _posts/2018-11-21-MeetingCpp-trip-report.markdown]: typo here (sorry): Ma**r**c

File: _posts/2018-11-21-MeetingCpp-trip-report.markdown [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+    <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</div>
+
+## Conclusion
+Meeting C++ is our favorite conference. It is a great event, with top speakers 
+and talks, well organized and run, in a great and interesting location. The talks 
+videos will be online, that will be great to be able to listen to the talks we 
+couldn’t attend. But the talks are not the best part of the conference. The people, 
+the networking, the spirit, the fun we had on the Quiz… difficult to explain.
```
Comment (uilianries) [inline _posts/2018-11-21-MeetingCpp-trip-report.markdown]: people > technology :heart:

File: _posts/2018-11-21-MeetingCpp-trip-report.markdown [hunk 4]
```diff
@@ -251,4 +251,4 @@ So looking forward to being there again next year.
 
 [^1]: Meeting Embedded took place on Wednesday, only Daniel and Diego were in Berlin to attend it.
 
-[^2]: On Thursday, more members of the Conan team joined from Spain: Luis and Javier.
+[^2]: On Thursday, more members of the Conan team joined from Spain: Luis and Javier. Also Katrin joined from the marketing team joined from France.
```
Comment (danimtb) [inline _posts/2018-11-21-MeetingCpp-trip-report.markdown]: repeated "joined"

File: _posts/2018-11-21-MeetingCpp-trip-report.markdown [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+        <img src="{{ site.url }}/assets/post_images/2018-11-21/conan-team.jpeg" width="600"/>
+        <figcaption>Luis, (The Frogarian), Katrin, Diego, Javi and Dani</figcaption>
+    </figure>
+</div>
+
+----
+
+[^1]: Meeting Embedded took place on Wednesday, only Daniel and Diego were in Berlin to attend it.
+
+[^2]: On Thursday, more members of the Conan team joined from Spain: Luis and Javier. Also Katrin joined from the marketing team joined from France.
```
Comment (danimtb) [inline _posts/2018-11-21-MeetingCpp-trip-report.markdown]:
```suggestion
[^2]: On Thursday, more members of the Conan team joined from Spain: Luis and Javier. Katrin from the marketing team also joined from France.
```

**Pull request merged**